packages/ember-views/lib/views/bound_component_view.js

```diff
@@ -0,0 +1,50 @@
+/**
+@module ember
+@submodule ember-views
+*/
+
+import { _Metamorph } from "ember-views/views/metamorph_view";
+import { read, chain, subscribe, unsubscribe } from "ember-metal/streams/utils";
+import { readComponentFactory } from "ember-views/streams/utils";
+import mergeViewBindings from "ember-htmlbars/system/merge-view-bindings";
+import EmberError from "ember-metal/error";
+
+export default Ember.ContainerView.extend(_Metamorph, {
```

Comment:
@igorT I don't think there is a parentView problem in the current implementation, unless I am misunderstanding. `BoundComponentView` is virtual, so it is skipped in the `parentView` getter.

![Console](https://dl.dropboxusercontent.com/s/po4waev0kacy3vu/2015-01-02%20at%2011.54%20AM.png?dl=0)

I'll add an assertion to make sure this remains true.